description: gcc6 needs explicit 'this->' for functions used in a lamba capturing 'this'
bug: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=67274
author: Michael Gilbert <mgilbert@debian.org>

--- a/ui/accessibility/ax_event_generator.cc
+++ b/ui/accessibility/ax_event_generator.cc
@@ -411,7 +411,7 @@ void AXEventGenerator::FireRelationSourc
         return;
 
       source_nodes.insert(source_node);
-      AddEvent(source_node, Event::RELATED_NODE_CHANGED);
+      this->AddEvent(source_node, Event::RELATED_NODE_CHANGED);
     });
   };
 
